CHANGELOG.md

```diff
@@ -5,6 +5,37 @@
 ## [Unreleased]
 
 
+## [0.5.0]
+
+_2023-09-20_
+
+### Spark
+
+* 🆕 Add IconButtons with all intents, shapes and sizes
+* 🆕 Add loading state to the IconButton component
+* 🆕 Add icon toggle button
+* 💄 Use M2 elevation system instead of the M3 one
+* ⬆️[Bump `compileSdk` and `targetSdk` to 34
+* 🎨 Change colors from LBC & KA
+* 🐛 Make the readonly Textfields not take the focus look when focused
+* 📝 Replace onliner with two distinct commands
+* 🐛 ModalFullScreenScaffold top padding issue when no illustration
+
+
+### Catalog App
+
+* 🆕 Add Icons demo to CatalogApp
+* 🆕 Add examples for tab component
+* 💄 Add Kleinanzeigen theme
+* 🎨 Add illustrations for components item
```

Comment:
Not trying to be too pushy, but are these illustrations being validated, or at least tracked somewhere to be updated later?

Reply:
Ah this was tracked in https://github.com/adevinta/spark-android/issues/714 but in the daily they were okay with these. At least they don't plan/have time to produce illustrations for each component for each demo apps

Reply:
ACK I was not aware of this 👍